src/main/java/org/entando/plugins/pda/core/model/Task.java

```diff
@@ -1,20 +1,20 @@
 package org.entando.plugins.pda.core.model;
 
-import java.util.Map;
-import lombok.Builder;
-import lombok.Data;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 
-@Data
-@Builder
-public class Task {
-    private String id;
-    private String name;
-    private String subject;
-    private String description;
-    private String status;
-    private Integer priority;
-    private Boolean skipable;
-    private String processId;
-    private String processInstanceId;
-    private Map<String,String> properties;
+public interface Task {
+    @JsonIgnore
+    Integer getId();
```

Comment:
Will the id always be able to be represented as Integer? Don't we have string ids? How about big numbers that can't fit into an integer?